src/index.jsx

```diff
@@ -20,6 +21,22 @@ subscribe(APP_READY, () => {
 });
 
 initialize({
+  handlers: {
+    config: () => {
+      mergeConfig({
+        BASE_URL: process.env.BASE_URL,
+        LMS_BASE_URL: process.env.LMS_BASE_URL,
+        LOGIN_URL: process.env.LOGIN_URL,
+        LOGOUT_URL: process.env.LOGOUT_URL,
+        CSRF_TOKEN_API_PATH: process.env.CSRF_TOKEN_API_PATH,
+        REFRESH_ACCESS_TOKEN_ENDPOINT: process.env.REFRESH_ACCESS_TOKEN_ENDPOINT,
+        DATA_API_BASE_URL: process.env.DATA_API_BASE_URL,
+        SECURE_COOKIES: process.env.NODE_ENV !== 'development',
+        SEGMENT_KEY: process.env.SEGMENT_KEY,
+        ACCESS_TOKEN_COOKIE_NAME: process.env.ACCESS_TOKEN_COOKIE_NAME,
```

Comment:
I have included the config settings that were previously in `config/index.js`. I did **not** pull in everything from `.env`, which contains a lot of unused settings.

I grepped for `process.env` and found two remaining usages:

* A few checks for `process.env.NODE_ENV == 'development'|'production'`. These feel like anti-patterns, but I assume it is OK to leave them as they were, since runtime config won't be overriding `NODE_ENV`.
* `<Footer logo={process.env.LOGO_POWERED_BY_OPEN_EDX_URL_SVG} />` -- This one confuses me, because AFAICT no other MFE has such a setting. Do other MFEs use the "Powered by Open edX" logo? If so, do they configure it some other way? I didn't see it in the default brand package. Anyway, I left this as-is, because tests failed when I tried to put it behind the `getConfig()` interface. Let me know if this seems like an issue Adolfo.

Reply:
The footer thing does indeed seem like an issue, but not a blocker here.

+1 for not including all the `.env` settings.

+1 for leaving `process.env.NODE_ENV` usages.